wordpress-mobile/WordPress-Android
# Feature/14933 reminders cancellation flow

Fixes #14933 

<img width=320 src="https://user-images.githubusercontent.com/990349/123381256-d9926d80-d5d3-11eb-8c7c-c0aa93784a8a.png"/>


To test:

- Turn on the blogging reminders flag (Me/App settings/Test feature configuration - BloggingRemindersFeatureConfig)
- Notice the blogging goals option appears (My Site > Settings > Blogging reminders)
- Click on for e.g. "Twice a week" button under Blogging goals setting
- Notice the blogging reminders prologue screen is shown
- Click on "Set goals"
- Deselect all days when screen is shown, then ensure "Update" button is enabled
- Click on "Update" button
- Notice the epilogue screen is shown. Done button is enabled, Clicking Done button should dismiss bottomsheet
- Go back to Settings and de-select all days and see that epilogue shows a "no reminders set" message


## Regression Notes
1. Potential unintended areas of impact
Site Settings

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Blogging reminders flow from Site Settings

3. What automated tests I added (or what prevented me from doing so)
Unit tests

PR submission checklist:
